ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -183,4 +184,53 @@ void sendUpdatedTransactionBuilder(
             @NotNull UtxoTransactionBuilder transactionBuilder,
             @NotNull FlowSession session
     );
+
+
+    /**
+     * Creates a query object for a named ledger query with the given name - if it exists. This query can be executed later.
+     * <p>
+     * Example usage:
+     * <ul>
+     * <li>Kotlin:<pre>{@code
+     * val query = utxoLedgerService.query("FIND_BY_TEST_FIELD", Int::class.java)
+     *     .setParameter("testField", "value")
+     *     .setParameter("participants", listOf("something"))
+     *     .setParameter("contractStateType", ContractState::class.java.name)
+     *     .setParameter("in-memory-filter-parameter", "parameter")
+     *     .setCreatedTimestampLimit(Instant.now())
+     *     .setOffset(0)
+     *     .setLimit(100)
+     * do {
+     *     val resultSet = query.execute()
+     *     processResultsWithApplicationLogic(resultSet.results)
+     *     query.setOffset(resultSet.newOffset)
+     * } while (resultSet.hasNextPage)
```

Comment:
We need to update the code docs due to the changes in the API.

```
do {
     *     val resultSet = query.execute()
     *     processResultsWithApplicationLogic(resultSet.results)
     *     query.setOffset(query.offset + 100)
     * } while (resultSet.results.isNotEmpty())
     *
```

Or something like that.